<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoviePlayerWindow</class>
 <widget class="QWidget" name="MoviePlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="pbPlayStop">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbPauseResume">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pbSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pbSaveAsImages">
     <property name="text">
      <string>Save as Images</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="cbLoopInfinitely">
     <property name="text">
      <string>Loop Infinitely</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Video</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblFrame">
        <property name="text">
         <string>Video</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbPlayStop</sender>
   <signal>clicked()</signal>
   <receiver>MoviePlayerWindow</receiver>
   <slot>playStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbPauseResume</sender>
   <signal>clicked()</signal>
   <receiver>MoviePlayerWindow</receiver>
   <slot>pauseResume()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSave</sender>
   <signal>clicked()</signal>
   <receiver>MoviePlayerWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSaveAsImages</sender>
   <signal>clicked()</signal>
   <receiver>MoviePlayerWindow</receiver>
   <slot>saveAsImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>playStop()</slot>
  <slot>pauseResume()</slot>
  <slot>save()</slot>
  <slot>saveAsImages()</slot>
 </slots>
</ui>
